src/main/java/org/dcsa/core/extendedrequest/QueryParameterParser.java

```diff
@@ -211,7 +211,7 @@ protected void parseSingleValueQueryParameter(QueryField queryField, String fiel
             }
         } else if (Temporal.class.isAssignableFrom(fieldType)) {
             String dateFormat = queryField.getDatePattern();
-            if (dateFormat != null) {
+            if (dateFormat != null && !dateFormat.equals("")) {

```

Comment:
Just  a suggestion, we can combine such checks in the future with `StringUtils.hasLength(dateFormat)` its a spring utility class.